x-pack/legacy/plugins/siem/public/components/toasters/utils.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
+import { isError } from 'lodash/fp';
+
+import { AppToast, ActionToaster } from './';
+import { isToasterError } from './errors';
+import { isApiError } from '../../utils/api';
+
+/**
+ * Displays an error toast for the provided title and message
+ *
+ * @param errorTitle Title of error to display in toaster and modal
+ * @param errorMessages Message to display in error modal when clicked
+ * @param dispatchToaster provided by useStateToaster()
+ */
+export const displayErrorToast = (
+  errorTitle: string,
+  errorMessages: string[],
+  dispatchToaster: React.Dispatch<ActionToaster>
+): void => {
+  const toast: AppToast = {
+    id: uuid.v4(),
```

Comment:
Since this uuid is a side effect I usually push it up and give it a default like so:

```ts
 export const displayErrorToast = (
  errorTitle: string,
  errorMessages: string[],
  dispatchToaster: React.Dispatch<ActionToaster>,
  id: uuid.v4(),
): void => {
```

So if someone wants to use the non-side effect version they can or if they want to push their own id down they can through tests, etc...

Not a biggie, just how I usually pull up uuid's when I can. I sometimes forget though, so just remind me if you see me not pulling them up ;-)